### DISCUSSION TITLE
Summary output options

### DISCUSSION THREAD

**Discussion by psyciknz** [jmbannon/ytdl-sub#1037] [Ideas]
is there any appetite for having a different format for the summary?  

I found this, and adapted for gotify: https://github.com/jmbannon/ytdl-sub/issues/709    

But I have a lot of commands in a script for removing the colour codes and having to reformat to get it to a json + markdown format .   Wondering if we had a feat request for a summary that is output as part of the command to a json file separately from the log.

So I've managed to get:
![image](https://github.com/user-attachments/assets/9797ce0d-04ad-43fe-8b47-bb0af5cfbe0a)

into gotify with the following: https://gist.github.com/psyciknz/80bf72d4c82af01de440bc435f4d877f

But I'm wanting a little prettier if possible. Ultimately I actually push it into tt-rss (I have tt-rss plugin I wrote for putting gotify content.  Which means I end up having to push markdown into gotify and then convert it to html for tt-rss (so annoying).  If I could get json from ytdl-sub it would make life a little easier for filtering etc.

In tt-rss it looks like the following:
<img width="482" alt="image" src="https://github.com/user-attachments/assets/78625dac-ed47-414d-a0c1-917a04c2134a">

**Comment by jmbannon** [comment]
Very cool! There is an old issue in the backlog to add JSON support in the transaction log: https://github.com/jmbannon/ytdl-sub/issues/746 Someday I'll get to it if someone doesn't beat me to it :slightly_smiling_face:
